charts/vsphere-cpi/v1.0.0/questions.yaml

```diff
@@ -1,4 +1,5 @@
 rancher_min_version: 2.5.6-rc1
+rancher_max_version: 2.5.6-rc99
```

Comment:
Added this so that we don't need to worry about this chart showing up in 2.5.6 before they are ready.